Opta-Stats/OptaStats.md

```diff
@@ -0,0 +1,82 @@
+# Opta Stats
+
+*Built by LATAM Team*
+
+**Supports:** *iOS and Android*
+*Current Version: 0.2.22 (Android), 1.0.52 (iOS)*
+
+## About
+The Opta Statistic Plugin is being developed primarily to show the statistics for Copa America 2019 tournament. Since Opta is the main provider for several events in LATAM we are pivoting this plugin to be general.
+
+## URL Schemes supported
+We defined a set of pre-configured number of screens which you can open through a URL scheme from anywhere in the application. In order to do so, we need to define a well-known URL schema. These are the proposed schemas for this integration.
+
+### URL Scheme to open a screen
+`<app_schema_id>://plugin?type=general&action=stats_open_screen&screen_id=<screen_id>&<other_data_key>=<other_data_value>`
+
+- `<app_schema_id>` **(Required)**: Application scheme id.
+- `<screen_id>` **(Required)**: A predefined `<screen_id>` from one of the following options:
+-- `home_screen`
+-- `match_details_screen`
+-- `all_matches_screen`
+-- `all_teams_screen`
+-- `team_screen`
+-- `player_screen`
+- `<other_data_key>` **(Required for specific screens)**: To show values from a specific entity:
+-- `match_id=<op_id>&push=<true_or_false>`
+-- `team_id` (`team_screen` and `all_matches_screen`)
+-- `player_id` (`player_screen`)
+
+#### Match details & Push option
+When opening match details screen through URL scheme, there is another key named push where indicates if the URL scheme comes from the push or if it’s being opened through the regular way.
+
+When the url scheme is coming from push the `match_id` is different from the one provided by Opta. Let's call this data, `match_id_from_push`. With this id provided by the push is necessary to send to Opta API as `match_id`, `urn:perform:opta:fixture:<match_id_from_push>`. The id coming from the push may have a letter at the beginning. It is necessary to remove this letter to use it as `match_id` in the API call.
```

Comment:
~With this id provided by the push is necessary to send to Opta API as `match_id`, `urn:perform:opta:fixture:<match_id_from_push>`.~ replace with: "The match id from the push must be sent to the Opta API as `match_id`, `urn:perform:opta:fixture:<match_id_from_push>`"